core/src/main/java/org/apache/druid/utils/CollectionUtils.java

```diff
@@ -75,9 +76,9 @@ public int size()
    * Returns a transformed map from the given input map where the value is modified based on the given valueMapper
    * function.
    */
```

Comment:
To the list of things to address in a follow-up PR: highlight the difference from `Maps.transformValues()` in the documentation, e. g. add a sentence like `Unlike {@link Maps#transformValues}, this method applies the mapping function eagerly to all key-value pairs in the source map and returns a new {@link HashMap}, while {@link Maps#transformValues} returns a lazy map view.`

Reply:
sure, will do. Thanks.